src/v2/Apps/Conversation/Components/Details.tsx

```diff
@@ -52,6 +54,13 @@ export const Details: FC<DetailsProps> = ({ conversation, ...props }) => {
   const item =
     conversation.items?.[0]?.item?.__typename !== "%other" &&
     conversation.items?.[0]?.item
+
+  const attachments = conversation.messagesConnection.edges
+    .map(({ node }) => node.attachments)
+    .filter(attachments => attachments.length > 0)
+    .reduce((previous, current) => previous.concat(current), [])
```

Comment:
Maybe I am missing something but is this `reduce` line needed? It seems it is iterating over an array and adding the items back to an empty array?

Reply:
The `reduce` function is being called on an array of arrays. `concat` has the interesting behavior in that if the input is an array it spreads it into the array it's being concatenated to (thereby flattening the array). Essentially the reduce is just taking the array of arrays and making it a flat array.